### PR TITLE
Prettier

### LINKED ISSUE #3: Let's add prettier?
I was very interested in this project. Do you mind if I add [.editorconfig](https://editorconfig.org/) and [prettier](https://prettier.io/) here?

### PR DESCRIPTION
**Pull request by demensky**
Fixes #3

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
```
Comment (demensky) [inline .prettierrc.json:3]: I suggest removing the indentation inside the curly braces `{}` by analogy with the square brackets `[]`.

```suggestion
    "trailingComma": "all",
    "bracketSpacing": false,
```

Comment (kosich) [reply .prettierrc.json:3]: Uhm, I'd prefer spaced `{}` :)

File: src/engine/DynamicEntry.ts [hunk 2]
```diff
@@ -20,34 +20,29 @@ export const DynamicEntry = () => {
...
-            )
+        if (
+            prev === NULL_CHILD_STUB ||
+            getType(prev) != getType(curr) ||
+            (isElement(curr) &&
+                isElement(prev) &&
+                (prev.type !== curr.type ||
+                    !Object.is(prev.props.key, curr.props.key) ||
+                    !Object.is(prev.props.xmlns, curr.props.xmlns) ||
+                    prev.props.children.length !== curr.props.children.length))
```
Comment (kosich) [inline src/engine/DynamicEntry.ts:45]: This reformatting makes me cry, but I'm ok to sacrifice it for greater good (tm) 🙂

**Comment by kosich** [comment]
LGTM
Let's keep `bracketSpacing: true` and merge

**Comment by demensky** [comment]
`bracketSpacing: true` is [the default behavior](https://prettier.io/docs/en/options.html#bracket-spacing).

**Comment by kosich** [comment]
Merged 🙂
Thanks for your contribution, Dmitry! 👍

**Pull request merged**